lib/chef-dk/skeletons/code_generator/recipes/build_cookbook.rb

```diff
@@ -119,6 +124,13 @@
     only_if "git status --porcelain |grep \".\""
   end
 
+  execute("git-add-delivery-project-toml") do
+    command("git add .delivery/project.toml")
+    cwd delivery_project_dir
+
+    only_if "git status --porcelain |grep \".\""
```

Comment:
Can you add a comment here explaining this? I can imagine this being pretty confusing for a new person jumping into this code.